### ISSUE TITLE
Where is your changelog?

### ISSUE THREAD

**Issue by OzzieOrca** [travist/jsencrypt#142]
I've looked at for it at these places and couldn't find it:
https://github.com/travist/jsencrypt/blob/master/CHANGELOG.md - (Like [React's Changelog.md](https://github.com/facebook/react/blob/master/CHANGELOG.md))
https://github.com/travist/jsencrypt/releases
Your website - http://travistidwell.com/jsencrypt/

https://yarnpkg.com/en/package/jsencrypt says you've published a beta and a RC this year. Wondering what those are and what is new/fixed. Thanks :)

**Comment by briantbutton** [comment]
Yeah.  I am in that camp with you.  3.0 looks a *lot* different - and better - than 2.3 (what I use now).  Trying to figure out if I want to upgrade . . .

**Comment by dartess** [comment]
@travist still looking for changelog
